src/redux/Graphql/actions.js

```diff
@@ -1,18 +1,41 @@
 const graphqlActions = {
   GET_TOPICS: 'GET_TOPICS',
   GET_TOPICS_RETURN: 'GET_TOPICS_RETURN',
-  getTopics: (filters, orderBy) => ({
+  getTopics: (filters, orderBy, limit, skip) => ({
     type: graphqlActions.GET_TOPICS,
     filters,
     orderBy,
+    limit,
+    skip,
+  }),
+  GET_MORE_TOPICS: 'GET_MORE_TOPICS',
```

Comment:
remove and use `getTopics()`. there's no difference in logic between `getTopics` and `getMoreTopics`